### PR TITLE
updated the image caption #14

### PR DESCRIPTION
**Pull request by Amit366**
Added the image caption.

### CODE REVIEW

**Comment by Amit366** [comment]
@iiitmjay I have updated it.

File: README.md [hunk 1]
```diff
@@ -21,7 +21,8 @@ The project deals with managing different layers of property on the basis of dif
 
 ## Road Map
 <p align="center">
-<img width="600" height="535" src="images/road_map.png"> 
+<img width="600" height="535" src="images/road_map.png">
+<figcaption>Road Map for Property Management System</figcaption>
```
Comment (IIITM-Jay) [inline README.md:25]: Amit @Amit366 the caption is not in the middle...
Can you try bringing it to the middle

**Review by IIITM-Jay — CHANGES_REQUESTED** [review]
@Amit366 try to bring it to the center of the page

**Comment by IIITM-Jay** [comment]
Amit @Amit366 if all the files have similar modification ... No need to push the same again and again

**Comment by IIITM-Jay** [comment]
Amit @Amit366 no need to push the same file for nearly 10 times, it will not look professional...
If something went wrong ask the mentor/ maintainer of the project repo for clarification

**Comment by Amit366** [comment]
And when I am trying to center it, it is moving to the right.

**Comment by IIITM-Jay** [comment]
> Sorry sir. Actually the Vscode stopped working thats why I am implementing the changes in the github itself.

No need to say sorry and try to learn and understand things... As the reviewers may also not find difficulties while reviewing..🤗

**Comment by Amit366** [comment]
> And when I am trying to center it, it is moving to the right.

Sir can you help me?

**Pull request closed without merging**